### PR TITLE
Fix blur effect in due to wrong popover position

### PR DESCRIPTION
**Pull request by d-go**
Fixed with a `Math.round()` the position of the popovers

### CODE REVIEW

File: src/clr-angular/popover/common/popover.spec.ts [hunk 1]
```diff
@@ -63,141 +63,141 @@ describe('Popover', function() {
...
     popoverInstance.anchor(anchor, Point.LEFT_CENTER, Point.RIGHT_BOTTOM);
     y = padding + anchorHeight / 2 - popoverHeight - popoverMargin;
-    expect(popover.style.transform).toEqual(`translateX(${x}px) translateY(${y}px)`);
+    expect(popover.style.transform).toEqual(`translateX(${Math.round(x)}px) translateY(${Math.round(y)}px)`);
 
     popover.style.transform = 'none';
     popoverInstance.anchor(anchor, Point.LEFT_TOP, Point.TOP_RIGHT);
     y = padding;
-    expect(popover.style.transform).toEqual(`translateX(${x}px) translateY(${y}px)`);
+    expect(popover.style.transform).toEqual(`translateX(${Math.round(x)}px) translateY(${Math.round(y)}px)`);
```
Comment (youdz) [inline src/clr-angular/popover/common/popover.spec.ts]: And this is why we need to have better test helpers. 😂 

I know it was just a simple search & replace, but it still feels bad.

**Comment by gnomeontherun** [comment]
Please squash and sign your commit, so we can merge. Thanks!

**Comment by gnomeontherun** [comment]
Hello @d-go, we cannot merge this PR unless you squash the commits into one, and sign-off in the comments. Otherwise it doesn't meet our contribution guidelines, and we'll end up redoing the PR on our end so we can close this out. 

Also it looks like our visual diff tests are failing, which means this change alters some of our reference images. That is somewhat expected, so you'd have to regenerate new ones by making sure you have Docker installed and running, then doing `npm run visual:fix set1 set2 set3 set4` to get new images.

**Comment by d-go** [comment]
Hi, I did squash them and signed-off that last commit (or at least I though so). I will try to sign-off those commits, and I was not aware about the images references, I will try to regenerate them

**Comment by gnomeontherun** [comment]
There are two commits in the branch, and currently neither one is signed-off. The commits have the same messages, which might be confusing. Also sign-off is not the same as gpg signed (which you did the last one), so read the comments above about doing the sign off by vmwclabot.

**Comment by d-go** [comment]
Got you

**Comment by d-go** [comment]
@gnomeontherun I tried to regenerate the image references with no success, I installed `gemini` and the `html-reporter` manually since they're not in the dev-dependencies (not sure if that's intended) but all the tests seem to fail and no image is updated/regenerated:
image

That is displayed for every test. Am I missing something?

**Comment by hippee-lee** [comment]
Hi @d-go - I'm not sure why docker was not able to connect to the server running the demos. 
If you haven't already tried to run it again, please do so. I have seen it time out like that the first time a system tries to run the Gemini suite. If its still giving you trouble I generated the gemini reference images here: https://github.com/vmware/clarity/commits/bug-popover-position
Feel free to cherry pick that commit into you PR branch if that is easier for you.

**Comment by hippee-lee** [comment]
To answer you gemini questions, yes it is intended for them to be globally installed. We do have a wiki page on CSS regression testing here: https://github.com/vmware/clarity/wiki/CSS-Regression-Testing.

**Comment by d-go** [comment]
Actually, I tried run it like several times but it didn't work, I got the same result over and over. Also I increased the WAIT_TIME to see if it was a matter of timeout but nothing. So I think  I'll do a cherry pick of your commit into my PR

**Comment by hippee-lee** [comment]
WAIT_TIME wouldn't help here. I believe the issue is that the chrome browser inside the docker container instance doesn't seem to be able to connect to the `lite-server` running on your hose. what OS are you on?

**Comment by d-go** [comment]
That makes sense... I'm running it on Windows 10

**Comment by hippee-lee** [comment]
Ok. Let me know if you are not able to cherry-pick the images I generated on the branch.

**Comment by hippee-lee** [comment]
@d-go - We really want to get this into the release this week so I cherrypicked your fix into a new branch and regenerated the gemini reference images so that you don't have to deal with the conflicts. 

The new PR will preserve the signed commit from you and add the images from me. 
If you would like I'll add your name to the contributors list on the Clarity website. 
Closing this PR so it doesn't get confused with this one: https://github.com/vmware/clarity/pull/2600

**Pull request closed without merging**